src/ISA/ISA/Regex.fs

```diff
@@ -56,17 +56,17 @@ module Pattern =
 
     /// Hits term accession, without id: ENVO:01001831
     [<LiteralAttribute>]
-    let TermAnnotationShortPattern = @"(?<idspace>\w+?):(?<localid>\w+)" //prev: @"[\w]+?:[\d]+"
```

Comment:
This is official vocabulary found in .obo file format specification

Reply:
Can you give a link?

Reply:
https://owlcollab.github.io/oboformat/doc/GO.format.obo-1_4.html#S.1.6